bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/item/EnrichedItemDTOMapper.java

```diff
@@ -134,6 +155,24 @@ private static EnrichedItemDTO map(Item item, ItemDTO itemDTO, boolean drillDown
         return enrichedItemDTO;
     }
 
+    private static String formatDateTime(Instant instant, ZoneId zoneId) {
+        String formatted = instant.atZone(zoneId).format(FORMATTER_TZ_RFC);
+        if (formatted.contains(".")) {
+            String sign = "";
+            if (formatted.contains("+")) {
+                sign = "+";
+            } else if (formatted.contains("-")) {
+                sign = "-";
+            }
+            if (!sign.isEmpty()) {
+                // the formatted string contains 9 fraction-of-second digits
+                // truncate at most 2 trailing groups of 000s
+                return formatted.replace("000" + sign, sign).replace("000" + sign, sign);
```

Comment:
With the formater mentioned above this should not be needed.

Reply:
See https://github.com/openhab/openhab-core/pull/3583#discussion_r1838883285